evm.md

```diff
@@ -1014,8 +1014,8 @@ Otherwise, it is calculated here using the "shortcut" formula used for running t
          <number>      HI     </number>
          <blockhashes> HASHES </blockhashes>
 
-    syntax Int ::= #blockhash ( List , Int , Int , Int ) [function]
- // ---------------------------------------------------------------
+    syntax Int ::= #blockhash ( List , Int , Int , Int ) [function, functional]
+ // ---------------------------------------------------------------------------
```

Comment:
I think the case `#blockhash(.List, 3, 3, 3)` is undefined?

Reply:
I removed the attribute for now. Let's wait for @dwightguth's response.